macros.ddoc

```diff
@@ -2,9 +2,11 @@ _=Fundamental macros that apply to all generated formats
 
 ARGS = $0
 COMMA = ,
-COMMENT = 
+COMMENT =
```

Comment:
This is spurious (due to my editor setup), but spaces at the end of the line are suspicious anyway.